docs/src/TextField.doc.js

```diff
@@ -210,43 +210,63 @@ card(
     id="tagsExample"
     name="Example: Tags"
     description={`
-    You can include \`Tag\` elements in the input using the \`tags\` prop.
-    Note that the \`TextField\` component does not internally manage tags. That should be handled in the application state through the component's event callbacks. This example creates new tags by splitting the input on spaces, commas, and semicolons, and removes them on backspaces.`}
+    You can include [Tag](/Tag) elements in the input using the \`tags\` prop.
+
+    Note that the \`TextField\` component does not internally manage tags. That should be handled in the application state through the component's event callbacks. We recommend creating new tags on enter key presses, and removing them on backspaces when the cursor is in the beginning of the field. We also recommend filtering out empty tags.
+
+    This example showcases the recommended behavior. In addition, it creates new tags by splitting the input on spaces, commas, semicolons.`}
```

Comment:
@ayeshakmaz feel free to recommend more changes, I'm not the best at writing these things

Reply:
Sounds great!